src/main/java/com/bob/jr/BobJr.java

```diff
@@ -179,34 +186,44 @@ public ServerResources setupPlayerAndCommands(final TextToSpeech tts, final Gate
         // create scheduler
         final TrackScheduler scheduler = new TrackScheduler(player, announcementPlayer, audioTrackCache);
 
+        // setup server resources
         final AudioProvider provider = new LavaPlayerAudioProvider(player, announcementPlayer);
         final ServerResources serverResources = new ServerResources(provider, scheduler, client, player, playerManager, tts, audioTrackCache);
+
+        // setup commands
+        setupBasicCommands(serverResources);
+        setupPlayerCommands(serverResources, playerManager, scheduler);
```

Comment:
I don't _love_ passing in the `playerManager` and `scheduler` here but I'm not sure if it's necessarily a bad thing.

Reply:
The `serverResources` object should actually have access to both `playerManager` and the `scheduler`. 
I'm using `ServerResources` to pass around "singletons" of all of those classes which is hacky as hell. Using proper dependency injection would be the clean way of doing it but we ain't have time for that.